src/coreclr/nativeaot/Runtime/MiscHelpers.cpp

```diff
@@ -416,3 +416,11 @@ EXTERN_C void QCALLTYPE RhCpuIdEx(int* cpuInfo, int functionId, int subFunctionI
     __cpuidex(cpuInfo, functionId, subFunctionId);
 }
 #endif
+
+#if defined(TARGET_X86) || defined(TARGET_ARM)
```

Comment:
I expect that wasm in runtimelab will want this too.

We do not need the ifdef around this. It should be fine to depend on the linker to get rid of it if it is unused.

Reply:
Done